src/components/vega-wallet/vega-wallet-container.tsx

```diff
@@ -23,7 +23,8 @@ export const VegaWalletFormContainer = () => {
     const run = async () => {
       if (!url) return;
       vegaWalletService.url = url;
-      const [version] = await vegaWalletService.getVersion();
+      const [version] = await vegaWalletService.getKeys();
+      console.log(version);
```

Comment:
Stray console.log here and variable name should change

```suggestion
      const [err] = await vegWalletService.getKeys();
```